src/components/figures/image/image.ts

```diff
@@ -0,0 +1,122 @@
+import { Component, useRef, useState } from "@odoo/owl";
+import { MENU_WIDTH } from "../../../constants";
+import { MenuItemRegistry } from "../../../registries/index";
+import { _lt } from "../../../translation";
+import { DOMCoordinates, Figure, SpreadsheetChildEnv, UID } from "../../../types";
+import { css } from "../../helpers";
+import { useAbsolutePosition } from "../../helpers/position_hook";
+import { Menu, MenuState } from "../../menu/menu";
+
+// -----------------------------------------------------------------------------
+// STYLE
+// -----------------------------------------------------------------------------
+css/* scss */ `
+  .o-image-menu {
```

Comment:
this css is duplicated with `.o-chart-menu`
=> extract it to a more generic `.o-figure-menu`